src/web/lib/components/StorageSpaceInformation/index.scss

```diff
@@ -5,6 +5,7 @@
   flex-direction: column;
   align-items: center;
   line-height: 1.5;
+  padding-top: 10px;
```

Comment:
This is shared by all tabs. To make sure that the gap is only added where we want it, remove this and add `margin-bottom: 10px;` to `.theme-colors-editor__list`. Note that the element itself already had `padding: 16px` at each side, but that failed to deliver the desired results because the content was clipped (with scrollbars).